File: python_backend/fenParser/getBoard/init_board.py
from custom_except import *

def init_board():
    """fenParser task 6, create empty board before know where the pieces are"""
    board = {}
    for y in range(1, 9):
        for x in range(1, 9):
            board[(x,y)] = "#"

    return board

if __name__ == "__main__":
    board = init_board()
    print(board)

File: app (1)/src/game_logic/fenParser/getBoard/initBoard.js
import {xyToRf} from "../../coordType/crdCnvrt";


export function initBoard() {
    /**fenParser task 6, get an empty board before we know where the pieces are*/
    let board = {}
    for (let y = 1; y < 9; y++) {
        for (let x = 1; x < 9; x++) {
            board[xyToRf(x, y)] = '#';
        }        
    }
    return board;
}

// module.exports = initBoard;

/** for node.js
if (require.main === module) {
    console.log(initBoard())

}
*/